README.md

```diff
@@ -28,10 +28,29 @@
 - [Extending Envoy with WASM and Rust](https://antweiss.com/blog/extending-envoy-with-wasm-and-rust/)
```

Comment:
I'm not sure if this should be in the README.md or a separate file linked from here, since it doesn't apply to consumers of the SDK?

Reply:
Either way SGTM. If you put it in a separate file, DEVELOPMENT.md would make sense. We don't currently have that in the C++ SDK but could update to match.